src/Standards/Squiz/Sniffs/Commenting/FunctionCommentThrowTagSniff.php

```diff
@@ -51,16 +57,18 @@ protected function processTokenWithinScope(File $phpcsFile, $stackPtr, $currScop
         $find[] = T_WHITESPACE;
 
         $commentEnd = $phpcsFile->findPrevious($find, ($currScope - 1), null, true);
-        if ($tokens[$commentEnd]['code'] === T_COMMENT) {
-            // Function is using the wrong type of comment.
-            return;
-        }
+        if ($this->skipFunctionsWithoutDocComment === true) {
+            if ($tokens[$commentEnd]['code'] === T_COMMENT) {
+                // Function is using the wrong type of comment.
+                return;
+            }
 
-        if ($tokens[$commentEnd]['code'] !== T_DOC_COMMENT_CLOSE_TAG
-            && $tokens[$commentEnd]['code'] !== T_COMMENT
```

Comment:
The second part of the condition has been removed as redundant, it's always true, because in line 61 we already return on opposite condition.